hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java

```diff
@@ -554,4 +573,8 @@ private void verifyPayerSignature(
             throw new PreCheckException(INVALID_SIGNATURE);
         }
     }
+
+    private boolean isSystemAccount(final Account account, final long systemAccountThreshold) {
+        return account.accountIdOrThrow().accountNumOrThrow() < systemAccountThreshold;
```

Comment:
It may be safer to use the following (so this doesn't throw for alias accounts):
```suggestion
        return account.accountIdOrElse(INVALID_ACCOUNT_ID).accountNumOrElse(-1L) < systemAccountThreshold;
```
That assumes the following definition.
```java
    private static final INVALID_ACCOUNT_ID =
            AccountID.newBuilder().shardNum(-1L).realmNum(-1L).accountNum(-1L).build();
```